x-pack/plugins/search_indices/public/components/quick_stats/quick_stats.tsx

```diff
@@ -68,7 +68,7 @@ export const QuickStats: React.FC<QuickStatsProps> = ({ index, mappings, indexDo
   const mappingStats = useMemo(() => countVectorBasedTypesFromMappings(mappings), [mappings]);
   const vectorFieldCount =
     mappingStats.sparse_vector + mappingStats.dense_vector + mappingStats.semantic_text;
-  const docCount = indexDocuments?.results._meta.page.total ?? 0;
```

Comment:
cc: @joemcelroy is there a reason we used `indexDocuments ` here instead of `index.documents`? I vaguely remember you iterating on this document count with several different approaches

Reply:
indexDocuments gets it from the search total which is just inaccurate for large amoutns

Reply:
Hey so I adjusted this to use indexDocuments which is configured to `track_total_hits:true` which is an exact count. 

index.documents has challenges:
- nested docs are counted separately: ie 1 doc using semantic_text field wont count as 1 which is confusing. 
- its async so uploading docs doesn't update correctly. This brings challenges with FTRs too.

Reply:
Ahhhh, interesting. The nested docs explains why I was seeing a massive discrepancy between the count on the indices list page and the specific index page.

Reply:
Alright let me remove the docs count and just do the a11y fix then